app/assets/javascripts/app.coffee.erb

```diff
@@ -0,0 +1,68 @@
+receta = angular.module('receta',[
+  'templates',
+  'ngRoute',
+  'controllers',
+])
+
+receta.config([ '$routeProvider',
+  ($routeProvider)->
+    $routeProvider
+    .when('/',
+      templateUrl: "<%= asset_path('index.html') %>"
+      controller: 'PcbuilderController'
+    )
+])
+
+components = [
```

Comment:
It is done. In the controller you can request /category/ to get all categories. Every category has a name. You can for example parse this to the front end in some sort of list structure.